ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -395,27 +395,29 @@ def osd_encryption_verification():
         raise UnsupportedFeatureError(error_message)
     osd_size = get_osd_size()
 
-    # Get "lsblk" command output on nodes where osd running
+    log.info("Get 'lsblk' command output on nodes where osd running")
     osd_node_names = get_osds_per_node()
     lsblk_output_list = []
     for worker_node in osd_node_names:
         lsblk_cmd = "oc debug node/" + worker_node + " -- chroot /host lsblk"
         out = run_cmd(lsblk_cmd)
         log.info(f"the output from lsblk command is {out}")
-        lsblk_output_list.append((out, len([worker_node])))
+        lsblk_output_list.append((out, len(osd_node_names[worker_node])))
 
-    # Verify "lsblk" command results are as expected
+    log.info("Verify 'lsblk' command results are as expected")
     for node_output_lsblk in lsblk_output_list:
         node_lsb = node_output_lsblk[0].split()
-        # Search 'crypt' in node_lsb list
+
+        log.info("Search 'crypt' in node_lsb list")
         all_occurrences_crypt = [
             index for index, element in enumerate(node_lsb) if element == "crypt"
         ]
-        # Verify all OSDs encrypted on node
+
+        log.info("Verify all OSDs encrypted on node")
         if len(all_occurrences_crypt) != node_output_lsblk[1]:
             raise EnvironmentError("OSD is not encrypted")
```

Comment:
It would be useful to have an info message confirming that newly added OSDs are encrypted.